doc/source/serve/http-servehandle.rst

```diff
@@ -110,146 +160,85 @@ You can follow the same pattern for other Starlette middlewares.
 
 .. _serve-handle-explainer:
 
-ServeHandle: Calling Endpoints from Python
-================================================
+ServeHandle: Calling Deployments from Python
+============================================
 
 Ray Serve enables you to query models both from HTTP and Python. This feature
 enables seamless :ref:`model composition<serve-model-composition>`. You can
-get a ``ServeHandle`` corresponding to an ``endpoint``, similar how you can
+get a ``ServeHandle`` corresponding to Deployment, similar how you can
 reach an endpoint through HTTP via a specific route. When you issue a request
-to an endpoint through ``ServeHandle``, the request goes through the same code
-path as an HTTP request would: choosing backends through :ref:`traffic
-policies <serve-split-traffic>` and load balancing across available replicas.
+to an endpoint through ``ServeHandle``, the request is load balanced across
+available replicas in the same way an HTTP request is.
 
-To call a Ray Serve endpoint from python, use :mod:`serve.get_handle <ray.serve.api.get_handle>` 
-to get a handle to the endpoint, then use 
-:mod:`handle.remote <ray.serve.handle.RayServeHandle.remote>` to send requests to that
-endpoint. This returns a Ray ObjectRef whose result can be waited for or retrieved using
-``ray.wait`` or ``ray.get``, respectively.
+To call a Ray Serve endpoint from python, use :mod:`Deployment.get_handle <ray.serve.api.Deployment>` 
+to get a handle to the Deployment, then use 
+:mod:`handle.remote <ray.serve.handle.RayServeHandle.remote>` to send requests
+to that Deployment. These requests can pass ordinary args and kwargs that are
+passed directly to the method. This returns a Ray ``ObjectRef`` whose result
+can be waited for or retrieved using ``ray.wait`` or ``ray.get``.
 
 .. code-block:: python
 
-    handle = serve.get_handle("api_endpoint")
-    ray.get(handle.remote(request))
-
-
-Accessing data from the request
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-When the request arrives in the model, you can access the data similarly to how
-you would with an HTTP request. Here are some examples how Ray Serve's built-in 
-``ServeRequest`` mirrors ```starlette.requests.request``:
```

Comment:
did we end up deleting `ServeRequest`?

Reply:
Deployments get regular args and kwargs now